components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -98,7 +98,7 @@ const BillingSubscriptions: React.FC = () => {
 
     useEffect(() => {
         getCloudSubscription()(dispatch, store.getState());
-        getCloudProducts()(dispatch, store.getState());
+        getCloudProducts(true)(dispatch, store.getState());
```

Comment:
nit: maybe defining a constant `const includeLegacyProducts = true;  -> getCloudProducts(includeLegacyProducts)(dispatch, store.getState()); ` will make the code self-documented and easier to read.